godot_fmod.cpp

```diff
@@ -28,7 +28,8 @@
 /*************************************************************************/
 
 #include "godot_fmod.h"
-#include "callbacks.h"
```

Comment:
I would keep the callback functions and data structures inside of a separate file just for cleanliness sake.

Reply:
Hi ! You're right. I get callbacks header back. it si imported by godot_fmod header so that EventInfo has access to CallbackInfo. I don't think EventInfo should go in callbacks namespace as it concerns more things.